### DISCUSSION TITLE
v5.38.0

### DISCUSSION THREAD

**Discussion by sokra** [webpack/webpack#13460] [General]
# Features

* `new URL("data:...", import.meta.url)` is now supported
* add `module.rules[].scheme` as condition to match the request scheme (like `data`, `http`, etc.)

# Bugfixes

* fix tracking of changes and removals during watching in some edge cases
* fix incorrect renaming of class fields in concatenatenated modules
* fix crash in HMR when removing runtimes from the compilation

# Performance

* lazy import some internal modules only when used
* allow unsafe caching of the entrypoint
* improve performance of sorting exports info map
* update to latest webpack-sources for improved source map performance

<hr /><em>This discussion was created from the release <a href='https://github.com/webpack/webpack/releases/tag/v5.38.0'>v5.38.0</a>.</em>

**Comment by phaistonian** [comment]
As of this release, webpack triggers a compilation twice.

That was also the case before this version, but using [this plugin](https://github.com/yessky/webpack-mild-compile) solved that issue.

After this version, this plugin no longer does the trick.

@sokra  Any pointers or ideas on what to try next?

**Reply by phaistonian** [reply]
@sokra Got it - this was caused by having `aggregateTimeout` set to 50 in `.watchOptions`.

Pushing it to 100 _fixed_ it.

Perhaps some kind of extra check is needed there?

**Reply by alexander-akait** [reply]
@phaistonian Do you use poll?

**Reply by phaistonian** [reply]
@alexander-akait  nope.

**Reply by alexander-akait** [reply]
weird, can you open an issue with reproducible test repo?
